supplementary/devices/index.rst

```diff
@@ -453,28 +454,9 @@ Devices
      - Navigation
      - Sensors
      - Other
-   * - :doc:`Sony Xperia L <taoshan>`
```

Comment:
You are still deleting taoshan related stuff. Please read the `git diff` next time before committing!